packages/flutter/test/cupertino/route_test.dart

```diff
@@ -328,4 +328,110 @@ void main() {
     expect(find.text('route'), findsOneWidget);
     expect(find.text('push'), findsNothing);
   });
+
+  testWidgets('Fullscreen route animates correct transform values over time', (WidgetTester tester) async {
+    await tester.pumpWidget(
+      CupertinoApp(
+        home: Container(
+          child: Builder(
+            builder: (BuildContext context) {
+              return CupertinoButton(
+                child: const Text('Button'),
+                onPressed: () {
+                  Navigator.push<void>(context, CupertinoPageRoute<void>(
+                    builder: (BuildContext context) {
+                      return Column(
+                        children: <Widget>[
+                          const Placeholder(),
+                          CupertinoButton(
+                            child: const Text('Close'),
+                            onPressed: () {
+                              Navigator.pop<void>(context);
+                            },
+                          )
+                        ],
+                      );
+                    },
+
+                    fullscreenDialog: true,
```

Comment:
just put this param on top so it's clearer what's being pushed

Reply:
Done